src/main/java/carpet/script/api/WorldAccess.java

```diff
@@ -877,14 +886,14 @@ else if (item instanceof TridentItem || item instanceof SwordItem)
 
             if (dropLoot)
             {
-                if (how < 0 || (tag != null && EnchantmentHelper.getLevel(Enchantments.SILK_TOUCH, tool) > 0))
+                if (fortuneLevel < 0 || (tag != null && EnchantmentHelper.getLevel(Enchantments.SILK_TOUCH, tool) > 0))
                 {
                     Block.dropStack(world, where, new ItemStack(state.getBlock()));
                 }
                 else
                 {
-                    if (how > 0)
-                        tool.addEnchantment(Enchantments.FORTUNE, (int) how);
+                    if (fortuneLevel > 0)
```

Comment:
honestly you would undo the blocks but can't undo the entities. Consider not track changes for harvest or destroy for that reason.

Reply:
Yeah good idea, I'm pretty sure those functions are meant for different things anyways